alpha/lib/model/Partner.php

```diff
@@ -1609,4 +1649,39 @@ public function getNotificationUrl()
 
 	public function getReferenceId() { return $this->getFromCustomData("referenceId", null); }
 	public function setReferenceId( $v ) { $this->putInCustomData("referenceId", $v); }
+
```

Comment:
Please revert all changes in partner object except for the getter and setter of GoogleOAuth2.